### DISCUSSION TITLE
PrivateGPT isntance is unable to summarize any document I give it

### DISCUSSION THREAD

**Discussion by KorruptedPineapple** [imartinez/privateGPT#1486] [Q&A]
Hello,

I'm new to AI development so please forgive any ignorance, I'm attempting to build a GPT model where I give it PDFs, and they become 'queryable' meaning I can ask it questions about the doc. 

I uploaded one doc, and when I ask for a summary or anything to do with the doc (in LLM Chat mode) it says things like 'I cannot access the doc, please provide one'.

Am I missing something?

**Comment by KorruptedPineapple** [comment]
![Screenshot 2024-01-04 201300](https://github.com/imartinez/privateGPT/assets/58124345/5a86034f-d520-41df-9a88-93287a4b50a5)

**Comment by KorruptedPineapple** [comment]
The next step is training it so it knows the right answers when given a new doc. But first this problem lol.

**Comment by felixxuss** [comment]
I am pretty sure that you have to switch to the "Query Docs" mode if you want the model to use the context from the file.
